src/app/login/page.tsx

```
@@ -0,0 +1,97 @@
+        {/* Login Form */}
+        <form className="space-y-4">
+          <div className="space-y-2">
+            <label className="text-sm text-gray-600">Email Address</label>
+            <Input
+              type="email"
+              className="border-lime-500 focus-visible:ring-lime-500"
+              required
+            />
+          </div>
+          <div className="space-y-2">
+            <label className="text-sm text-gray-600">Account Password</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)